easybuild/easyblocks/f/fdtd_solutions.py

```diff
@@ -33,64 +33,45 @@
 from easybuild.easyblocks.generic.packedbinary import PackedBinary
 from easybuild.easyblocks.generic.rpm import rebuild_rpm
 from easybuild.tools.build_log import EasyBuildError
-from easybuild.tools.run import run_cmd_qa
+from easybuild.tools.filetools import rmtree2
+from easybuild.tools.run import run_cmd, run_cmd_qa
 
 
 class EB_FDTD_underscore_Solutions(PackedBinary):
     """Support for building/installing FDTD Solutions."""
 
-    def build_step(self):
-        """No build step for FDTD Solutions."""
-        # locate RPM and rebuild it to make it relocatable
-        rpms = glob.glob(os.path.join(self.cfg['start_dir'], 'rpm_install_files', 'FDTD-%s*.rpm' % self.version))
+    def extract_step(self):
+        """
+        After unpacking the main tar file, we need to unpack the rpm
+        inside it.
+        """
+        super(EB_FDTD_underscore_Solutions, self).extract_step()
+
+        rpms = glob.glob(os.path.join(self.src[0]['finalpath'], 'rpm_install_files', 'FDTD-%s*.rpm' % self.version))
         if len(rpms) != 1:
             raise EasyBuildError("Incorrect number of RPMs found, was expecting exactly one: %s", rpms)
-        rebuilt_dir = os.path.join(self.cfg['start_dir'], 'rebuilt')
-        rebuild_rpm(rpms[0], rebuilt_dir)
+        cmd = "rpm2cpio %s | cpio -idm " % rpms[0]
+        run_cmd(cmd, log_all=True, simple=True)
 
-        # replace original RPM with relocatable RPM
-        rebuilt_rpms = glob.glob(os.path.join(rebuilt_dir, '*', '*.rpm'))
-        if len(rebuilt_rpms) != 1:
-            raise EasyBuildError("Incorrect number of rebuilt RPMs found, was expecting exactly one: %s", rebuilt_rpms)
+    def build_step(self):
+        """No build step for FDTD Solutions."""
+        pass
 
+    def install_step(self):
+        """Install FDTD Solutions using copy tree."""
         try:
-            os.rename(rpms[0], '%s.bk' % rpms[0])
-            shutil.copy2(rebuilt_rpms[0], rpms[0])
+            # shutil.copytree doesn't allow the target directory to exist already
+            rmtree2(self.installdir)
+            shutil.copytree(os.path.join(self.cfg['start_dir'], 'opt', 'lumerical', 'fdtd'),
```

Comment:
@akesandgren This should be updated to use `copy_dir` from `filetools` (no need for the `try`/`except` anymore then)

Also, please define a variable with the path you're copying, since you're using it in two places (here + the error message)

```python
ftdt_dir = os.path.join(self.cfg['start_dir'], 'opt', 'lumerical', 'fdtd')
copy_dir(ftdt_dir, self.installdir, symlinks=self.cfg['keepsymlinks'])